internal/oci/runtime_oci.go

```diff
@@ -630,6 +650,12 @@ func (r *runtimeOCI) StopContainer(ctx context.Context, c *Container, timeout in
 		return err
 	}
 
+	if c.Spoofed() {
+		c.state.Status = ContainerStateStopped
+		c.state.Finished = time.Now()
```

Comment:
Do we guard at upper level so that we don't update Finished once it is set?

Reply:
above this we check ShouldBeStopped(), which verifies the Status is not ContainerStateStopped

Reply:
What happens if crio restarts after this point?

Reply:
that's a good point.

In this version, I've allowed spoofed containers to have their state written and read from disk in ContainerState{To,From}Disk. that will allow the appropriate state to be preserved across cri-o restarts